ansible/files/whisks_design_document_for_activations_db.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
note/question to self: how will this work if _split_ is false since both design docs have the same name?